src/com/can/CMakeLists.txt

```diff
@@ -9,44 +9,51 @@
 # SPDX-License-Identifier: EPL-2.0
 # 
 # Contributors:
-#     Malte Grave - initial API and implementation and/or initial documentation
+#     Malte Grave, Alexander Trojnin - initial API and implementation and/or initial documentation
 # *******************************************************************************/
 # *******************************************************************************/
 #############################################################################
 # CAN Com Layer (currently only usable on Linux)
 #############################################################################
 
-function(CHECK_IF_KERNEL_MODULE_IS_LOADED)
-  execute_process(
-    COMMAND lsmod
-    COMMAND grep -wE "^(can|can_raw|vcan|can_dev)"
-    COMMAND awk "{print $1}"
-    COMMAND tr "\n" " "
-    OUTPUT_VARIABLE CAN_MODULES_FOUND
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-  )
-  message(STATUS "Found CAN modules: ${CAN_MODULES_FOUND}")
+forte_add_network_layer(CAN OFF "can" CCANComLayer CANLayer "Enable the CAN Commuication Layer")
+set(FORTE_COM_CAN_MODULE_CHECK OFF CACHE BOOL "Build Tests")
```

Comment:
I think it would be more appropriate to check the modules if the CAN layer is active, like before. Also, the String "Build Tests" seems a bit off

Reply:
"On" default. I Don't checking CAN modules because need cross-build for embedded system. The device has 32MB of memory, and gcc is the least I'd like to see there.

Reply:
Ah okay, I see the test makes sense when it comes to cross-builds.